chia/cmds/plots.py

```diff
@@ -194,3 +194,11 @@ def remove_cmd(ctx: click.Context, final_dir: str):
 @click.pass_context
 def show_cmd(ctx: click.Context):
     show_plots(ctx.obj["root_path"])
+
+
+@plots_cmd.command("plotinfo", short_help="get info from a plot file")
+@click.option("-f", "--file", help="plot file", type=Path)
```

Comment:
specifying a file is mandatory, isn't it? if so, it would seem reasonable to make it a positional argument instead. If you want to specify more than one, do you really need `-f` in front of each file?

Reply:
Sorry, I just noticed that. I'm going to test my code. The headache is that a real plot file is so large that I don't know how to test this method. Is there any better way?

Reply:
you can create a small plot file, k=21 or so. We do this already in CI as a smoke test